stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md

```diff
@@ -0,0 +1,45 @@
+# Exceptions Overview
+
+## Agenda
++ [What is an Exception?](#what-is-an-exception)
++ [The Three Kinds of Exceptions](#the-three-kinds-of-exceptions)
++ [Materials](#materials)
+
+## <a id="what-is-an-exception"></a> What is an Exception? 
```

Comment:
Could you wrap header with tag `<a>`? For example
`<a id="what-is-an-exception">What is an Exception?</a>`

Reply:
yes, thanks